client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -145,17 +145,31 @@ export default React.createClass( {
 	},
 
 	getUpdateWarning() {
-		const newVersion = this.getAvailableNewVersion();
-		if ( newVersion ) {
+		const newVersions = this.getAvailableNewVersions();
+		if ( newVersions.length > 0 ) {
+			if ( this.props.selectedSite ) {
+				return (
+					<Notice
+						status="is-warning"
+						className="plugin-meta__version-notice"
+						showDismiss={ false }
+						icon="sync"
+						text={ i18n.translate( 'A new version is available.' ) }>
+						<NoticeAction onClick={ this.handlePluginUpdatesSingleSite }>
+							{ i18n.translate( 'Update to %(newPluginVersion)s', { args: { newPluginVersion: newVersions[ 0 ] } } ) }
+						</NoticeAction>
+					</Notice>
+				);
+			}
 			return (
 				<Notice
 					status="is-warning"
 					className="plugin-meta__version-notice"
 					showDismiss={ false }
 					icon="sync"
-					text={ i18n.translate( 'A new version is available.' ) }>
-					<NoticeAction onClick={ this.handlePluginUpdates }>
-						{ i18n.translate( 'Update to %(newPluginVersion)s', { args: { newPluginVersion: newVersion } } ) }
+					text={ i18n.translate( 'A new version is available in some of your sites' ) }>
```

Comment:
I think it would be great if we could return a more specific like. How many sites do we need to update to the latest version or the title of the site.